components/admin_console/team_channel_settings/channel/details/channel_details.tsx

```diff
@@ -43,7 +43,7 @@ export interface ChannelDetailsProps {
     groups: Group[];
     totalGroups: number;
     allGroups: Dictionary<Group>;
-    channelPermissions?: ChannelPermissions[];
+    channelPermissions: ChannelPermissions[];
```

Comment:
This was always provided except by unit tests, so I added it to the test props, made it non-optional, and removed a bunch of not-null assertions to make this code easier to read.